lib/core/src/lib/comments/comments.component.html

```diff
@@ -29,8 +29,8 @@
             </div>
     </div>
 
-    <div *ngIf="comments.length > 0">
+    <ng-container *ngIf="comments.length > 0">
```

Comment:
Do we need that ng-container at all? It contains single element so no need for container as we can probably just add ngIf on adf-comment-list